pkg/csi/service/vanilla/controller.go

```diff
@@ -423,7 +433,7 @@ func (c *controller) ReloadConfiguration() error {
 
 		c.manager.VolumeManager.ResetManager(ctx, vcenter)
 		c.manager.VcenterConfig = newVCConfig
-		c.manager.VolumeManager = cnsvolume.GetManager(ctx, vcenter, operationStore, true)
+		c.manager.VolumeManager = cnsvolume.GetManager(ctx, vcenter, operationStore, true, false)
```

Comment:
Confirming, is this false because we are yet to implement Multi VC for `ReloadConfiguration()`?

Reply:
ReloadConfiguration will be implemented in the later PR. We have a separate task for that.